index.html

```diff
@@ -1,21 +1,16 @@
 <!DOCTYPE html>
 <head>
-  <title>TOP POT DONUT DEMAND SIMULATOR</title>
+  <link href="css/style.css" type="text/css" rel="stylesheet" />
+  <link href='http://fonts.googleapis.com/css?family=Fjalla+One|Indie+Flower' rel='stylesheet' type='text/css'>
   <meta charset="UTF-8"/>
-    <meta http-equiv="X-UA-Compatible" content="IE=Edge"/>
-    <style type="text/css">
-      body {
-        background-image: url("img/2371ff_homer-donut.gif");
-        background-position: right;
+  <meta http-equiv="X-UA-Compatible" content="IE=Edge"/>
 
-
-        padding: 10px;}
-    </style>
- <link href="css/style.css" type="text/css" rel="stylesheet" />
+  <title>Top Pot Donut Demand Simulator</title>
 
 </head>
 <body>
-  <h2><font color="white">TOP POT DONUT DEMAND SIMULATOR </font></h2>
+  <h2><font color="white">Top Pot Donut Demand Simulator </font></h2>
+</style>
```

Comment:
What's up with this style tag? Also, the Font tag on line 12 should not be here. That's stylistic, and should be in your CSS.